SIPs/sip_001.md

```diff
@@ -0,0 +1,172 @@
+# SPOOL Improvement Proposal 001
```

Comment:
I would just use `1` like python does with PEP: https://www.python.org/dev/peps/

Otherwise, why not `00001`, to emphasize that SPOOL is for the long term, in the same way that the [Long Now Foundation](http://longnow.org/) works, e.g.: it was founded in 01996 ... see http://longnow.org/about/ for instance, or why not just 1?